packages/paste-core/components/button-group/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import {render, screen} from '@testing-library/react';
+import {Button} from '@twilio-paste/button';
+import {ButtonGroup} from '../src';
+import {CustomizationProvider} from '@twilio-paste/customization';
+import {Attached, Unattached} from '../stories/index.stories';
+
+describe('ButtonGroup', () => {
+  it('should render a group visibly into document', () => {
+    render(<Unattached />);
+    const btnGroup = screen.getByRole('group');
+    expect(btnGroup).toBeVisible();
+  });
+
+  it('should render children visibly into document', () => {
+    render(<Unattached />);
+    const btns = screen.getAllByRole('button');
+    expect(btns).toHaveLength(3);
+    for (const btn of btns) {
+      expect(btn).toBeVisible();
+    }
+  });
+
+  it('should accept attached prop and apply styles', () => {
+    render(<Attached />);
+    const middleBtn = screen.getAllByRole('button')[1];
+    expect(middleBtn).toHaveStyle({borderRadius: 0});
+  });
+
+  it('should default to unattached and apply styles', () => {
+    render(<Unattached />);
+    const middleBtn = screen.getAllByRole('button')[1];
+    expect(middleBtn).not.toHaveStyle({borderRadius: 0});
```

Comment:
question: why are you testing that the middle button doesn't have border radius 0 instead of testing that it does have border radius 4 px?